src/content/reference/react/useDeferredValue.md

```diff
@@ -501,25 +501,25 @@ input { margin: 10px; }
 
 <DeepDive>
 
-#### How does deferring a value work under the hood? {/*how-does-deferring-a-value-work-under-the-hood*/}
+#### Comment délayer une valeur fonctionne-t-il en sous-jacent ? {/*how-does-deferring-a-value-work-under-the-hood*/}
```

Comment:
```suggestion
#### Comment une valeur différée fonctionne-t-elle sous le capot ? {/*how-does-deferring-a-value-work-under-the-hood*/}
```

Reply:
Tiens, qu'est-ce qui te gênait ici ? "Sous le capot" est la trad établie (et littérale) de "under the hood", et on l'utilise régulièrement dans ces docs (y compris l'édition précédente).